apps/lend/src/entities/token/lib/last-results.ts

```diff
@@ -0,0 +1,7 @@
+import { queryClient } from '@/shared/api/query-client'
+import { tokenKeys } from '@/entities/token'
+
+export const getLastTokenUsdRate = (chainId: ChainId, tokenAddress: string) => queryClient.getQueryData<number>(tokenKeys.usdRate({
```

Comment:
The reason I wanted to keep this function isolated is that it should be temporary. It's a bit of an anti-pattern to use the last value instead of using the query.

Reply:
No, I think it's fine, Tanstack even advises it. You have no other way to get the value inside an API function other than using that function or an asynchronous one to get the value precisely.